main/src/test/java/cgeo/geocaching/connector/gc/ImportBookmarkLinksTest.java

```diff
@@ -0,0 +1,64 @@
+package cgeo.geocaching.connector.gc;
+
+import org.junit.Test;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class ImportBookmarkLinksTest {
```

Comment:
Kudos and thumbs up for adding a test class!

Reply:
Thank you :)